Packs/Box/Integrations/BoxEventsCollector/BoxEventsCollector.py

```diff
@@ -74,17 +74,42 @@ class Config:
         validate_assignment = True
 
 
+def not_gate(v):
+    """Due to a bug in the validator object (collision with CommonServerPython)
+    we can pass this a a simple lambda. So here we are.
+
+    Just doing not if v is bool, else it is true.
+
+    Used when getting insecure and should change the insecure to verify.
+    insecure == True, verify is False.
+    """
+    v_ = parse_obj_as(bool, False if v is None else v)
+    return not v_
+
+
 class BoxEventsRequestConfig(IntegrationHTTPRequest):
     # Endpoint: https://developer.box.com/reference/get-events/
     url = parse_obj_as(AnyUrl, 'https://api.box.com/2.0/events')
```

Comment:
Make `api.box.com` an advanced parameter, and use everywhere.

Reply:
what if they want to use `test.api.box.com` or `beta-tests.api.box.com`?
Box could offer this service to their customers.

Reply:
Don't we support the official API and that's it?